diesel/src/query_builder/select_statement/dsl_impls.rs

```diff
@@ -196,3 +200,58 @@ impl<'a, S, F, D, W, O, L, Of, G, DB> InternalBoxedDsl<'a, DB>
         )
     }
 }
+
+impl<'a, F, D, W, O, L, Of, G, DB> InternalBoxedDsl<'a, DB>
```

Comment:
Also @nrc if you want a pathological case for rustfmt discussions, see the code 2 comments up.